src/tracks/SampleTrack.cpp

```diff
@@ -310,11 +314,18 @@ void SampleTCO::loadSettings( const QDomElement & _this )
 		useCustomClipColor( true );
 		setColor( _this.attribute( "color" ) );
 	}
+
+	if( _this.hasAttribute( "reversed" ) )
+	{
+		m_sampleBuffer->setReversed( true );
+		emit wasReversed();		// tell SampleTCOView to update the view
+	}
```

Comment:
I'd like to point out why I didn't just call `reverseSample()` over here. For one, it had bugs and was unable to consistently load the sample reversed, and `reverseSample()` also sets the modified status, which we don't want during loading ideally.

Reply:
I'd have to check it further, but if I'm not mistaken there's a variable that is set while the project is being loaded. Maybe the `reverseSample()` method could check for that variable and not call `Engine::getSong()->setModified();` if it's true. It sounds totally fine to do it the way you're doing here though, just something to maybe consider!

Reply:
Yep, but the bug issue still does stand. I tried to work around it and see what caused it but I wasn't able to come to a conclusion, so I decided to just set the variable here.

In case you want to try and replicate: even though `reverseSample()` is called, it still unreverses it to its original way (or doesn't reverse it at all). This is at load time.

Reply:
Oh, I just noticed `reverseSample` is a TCO view method. So I think that's right, you'll have to call the SampleBuffer method `setReversed` instead because the TCO view might not even have been created yet I think. Just a guess, I haven't looked much further, but seems like the way it's done now is the right way!